crates/communication/src/server/parameters/subscriptions.rs

```diff
@@ -0,0 +1,1359 @@
+        //         }))
+        //     ),
+        //     "unexpected {response:?}",
+        // );
+        // match response_receiver.try_recv() {
+        //     Err(TryRecvError::Empty) => {}
+        //     response => panic!("unexpected result from try_recv(): {response:?}"),
+        // }
+        // assert_eq!(
+        //     *subscribed_outputs_reader.next(),
+        //     HashSet::from_iter([path.clone()]),
+        // );
+
+        // outputs_changed.notify_one();
+        // let subscribed_data = response_receiver.recv().await.unwrap();
+        // assert_eq!(
+        //     subscribed_data,
+        //     Response::Textual(TextualResponse::Outputs(
+        //         TextualOutputsResponse::SubscribedData {
+        //             items: [(
+        //                 SUBSCRIPTION_ID,
+        //                 TextualDataOrBinaryReference::TextualData { data: value }
+        //             )]
+        //             .into()
+        //         }
+        //     )),
+        // );
+        // match response_receiver.try_recv() {
+        //     Err(TryRecvError::Empty) => {}
+        //     response => panic!("unexpected result from try_recv(): {response:?}"),
+        // }
+        // assert_eq!(
+        //     *subscribed_outputs_reader.next(),
+        //     HashSet::from_iter([path]),
+        // );
+
+        // request_sender
+        //     .send(ClientRequest {
+        //         request: OutputsRequest::Unsubscribe {
+        //             id: 1337,
+        //             subscription_id: SUBSCRIPTION_ID,
+        //         },
+        //         client: Client {
+        //             id: client_id,
+        //             response_sender: response_sender.clone(),
+        //         },
+        //     })
+        //     .await
+        //     .unwrap();
+        // let response = response_receiver.recv().await.unwrap();
+        // assert!(
+        //     matches!(
+        //         response,
+        //         Response::Textual(TextualResponse::Outputs(
+        //             TextualOutputsResponse::Unsubscribe {
+        //                 id: 1337,
+        //                 result: Ok(()),
+        //             }
+        //         ))
+        //     ),
+        //     "unexpected {response:?}",
+        // );
+        // match response_receiver.try_recv() {
+        //     Err(TryRecvError::Empty) => {}
+        //     response => panic!("unexpected result from try_recv(): {response:?}"),
+        // }
+        // assert_eq!(*subscribed_outputs_reader.next(), HashSet::new());
+
+        // outputs_changed.notify_one();
+        // match response_receiver.try_recv() {
+        //     Err(TryRecvError::Empty) => {}
+        //     response => panic!("unexpected result from try_recv(): {response:?}"),
+        // }
+        // assert_eq!(*subscribed_outputs_reader.next(), HashSet::new());
+
+        // drop(request_sender);
+        // provider_task.await.unwrap();
```

Comment:
Is this still relevant?
Pls remove.

Reply:
This was a residual from porting test cases for the subscriptions task. Removed.